ftw/inflator/creation/sections/resolvepath.py

```diff
@@ -0,0 +1,50 @@
+from collective.transmogrifier.interfaces import ISection
+from collective.transmogrifier.interfaces import ISectionBlueprint
+from collective.transmogrifier.utils import defaultMatcher
+from ftw.inflator.creation.sections.helpers import map_recursive
+from zope.interface import classProvides
+from zope.interface import implements
+
+
+class ResolvePath(object):
+
+    PREFIX = 'resolvePath::'
+
+    classProvides(ISectionBlueprint)
+    implements(ISection)
+
+    def __init__(self, transmogrifier, name, options, previous):
+        self.transmogrifier = transmogrifier
+        self.context = transmogrifier.context
+        self.previous = previous
+        self.pathkey = defaultMatcher(options, 'path-key', name, 'path')
+
+    def __iter__(self):
+        for item in self.previous:
+            yield map_recursive(
+                lambda value: value.startswith(self.PREFIX),
+                lambda value: self._resolve(item, value),
+                item)
+
+    def _resolve(self, item, value):
+        value = value.lstrip(self.PREFIX)
```

Comment:
Do not use ``lstrip`` with more than one characters because it usually does not do what you want:
```python
>>> 'resolvePath::/foo/bar'.lstrip(str(sorted('resolvePa:th')))
'/foo/bar'
```
:wink: